pkg/controller/resourcegraphdefinition/controller_status.go

```diff
@@ -183,3 +186,103 @@ func newGraphVerifiedCondition(status metav1.ConditionStatus, reason string) v1a
 func newCustomResourceDefinitionSyncedCondition(status metav1.ConditionStatus, reason string) v1alpha1.Condition {
 	return v1alpha1.NewCondition(v1alpha1.ResourceGraphDefinitionConditionTypeCustomResourceDefinitionSynced, status, reason, "Custom Resource Definition is synced")
 }
+
+// reconcileStatus updates the status of the RGD instance based on the status of its child resources
+func (r *ResourceGraphDefinitionReconciler) reconcileStatus(ctx context.Context, instance *v1alpha1.ResourceGraphDefinition) error {
```

Comment:
An RGD doesn't have children, in fact i'm not sure if we should expose this kind of information in the RGD it self.